lib/sidekiq/cron.rb

```diff
@@ -2,7 +2,6 @@
 require "sidekiq/cron/namespace"
 require "sidekiq/cron/poller"
 require "sidekiq/cron/launcher"
-require "sidekiq/cron/schedule_loader"
```

Comment:
I removed it to avoid a cyclic require issue. It'll be great if you could share best solution for it. 
If I add it again, I'm facing `undefined method `configuration' for Sidekiq::Cron:Module`.

Reply:
Ok, I understand, but we'll need to find a solution. Otherwise the automatic scheduler loader is not going to work anymore. Maybe we can move the `Configuration` class outside the main module 🤔?

Reply:
Yes, I'll have a look tonight. If you've any suggestions please let me know.

Reply:
The point is that we need to run the default config, before loading other things. So another idea is to move these `require`s at the end of the file, after running the `Sidekiq::Cron.configure` (which will create the default config).

Reply:
Yeah, that's a quick idea that came out of my head.

Reply:
It'll still cause cyclic require issue, right?